requirements/docker.txt

```diff
@@ -10,7 +10,7 @@
     # via
     #   -r requirements/base.in
     #   -r requirements/docker.in
-gevent==22.10.2
+gevent==23.9.1
```

Comment:
If we're bumping versions the bounds should be updated in `setup.py` as this ensures that the correct versions of `gevent` (or similar) will be installed irrespective of how people build Superset.

Reply:
This is only included on docker, we have no references of `gevent` on `setup.py`

Reply:
Maybe we should add "gevent" as an optional `extras_require`?

Reply:
ok, I don't think we interact or will colide with `gevent` in any way since it could be used with `gunicorn`. `gunicorn` does not set any upper boundaries so I'm doing the same